providers/flagd/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/open-feature/go-sdk-contrib/providers/flagd
 
-go 1.21
+go 1.22.0
```

Comment:
Is this version bump required? If so, can it be done in a separate PR?

Reply:
let me have a look seems like my local ide

Reply:
done revert the version bump